doc/user_guide/checkers/features.rst

```diff
@@ -734,7 +734,9 @@ Nonascii-Checker checker Messages
   interoperability. Further reading: -
   https://peps.python.org/pep-0489/#export-hook-name -
   https://peps.python.org/pep-0672/#confusing-features -
-  https://bugs.python.org/issue20485
+  https://bugs.python.org/issue20485 -
+  https://github.com/python/cpython/issues/64684#issuecomment-1093644763 -
+  https://terpconnect.umd.edu/~zben/Web/CharSet/htmlchars.html
```

Comment:
Everything starting from "further reading" should go in ``related.rst`` imo (and be removed from the checker's code).

Reply:
Ok, reverting